File: src/services/order.service.js
import axios from "axios";

const API_URL="http://localhost:8083/sys/order";

class OrderService{

    addOrder(order)
    {
        return axios.post(API_URL+"/postorder",order);
    }

    findAll(order)
    {
        return axios.get(API_URL+"/getorder",order);
    }

    deleteorder(id)
    {
        return axios.delete(API_URL + "/orders/" + id);
    } 
    
    addOrderFood(order)
    {
        return axios.post(API_URL+"/postorder",order);
    }
    
}

export default new OrderService; 

File: src/main/java/com/SYNTIARO_POS_SYSTEM/Service/OrderService.java
package com.SYNTIARO_POS_SYSTEM.Service;



import com.SYNTIARO_POS_SYSTEM.Entity.Bill;
import com.SYNTIARO_POS_SYSTEM.Entity.OrderFood;
import com.SYNTIARO_POS_SYSTEM.Entity.Orders;
import com.SYNTIARO_POS_SYSTEM.Response.OrderResponse;
import org.springframework.http.ResponseEntity;

import java.time.LocalDate;
import java.util.List;
import java.util.Optional;

public interface OrderService {
    List<Orders> getorder();
    int addOrder(Orders order);
    Orders updateOrder(Orders order);
    void deleteorder(int i);

    Optional<Orders> getorderbyid(Integer orderid);

    // fetch by storeid
    List<Orders> getOrdersByStoreId(String storeId);

    public Orders placeOrders(OrderFood orderFood);





}
